tests/v2/test_lint.py

```diff
@@ -39,6 +39,31 @@ def test_check_incompatible_targets():
     assert "overlapping targets {'p1'}" in error.message
 
 
+def test_validate_initial_change_symbols():
+    """Test validation of symbols in target value expressions for changes
+    applied at the start of an experiment."""
+    problem = Problem()
+    problem.model = SbmlModel.from_antimony("p1 = 1; p2 = 2")
+    problem.add_experiment("e1", 0, "c1", 1, "c2")
```

Comment:
Unrelated: the docstring for `add_experiment` is currently missing that multiple conditions applied at a single timepoint are specified as a list inside `args`

https://github.com/PEtab-dev/libpetab-python/blob/8795dbfe4c5cf42d9f9eae2e88f4b536810cf45d/petab/v2/core.py#L2100-L2109